app/workers/education_form/create_daily_spool_files.rb

```diff
@@ -46,7 +46,7 @@ def group_submissions_by_region(records)
     def write_files(sftp: nil, structured_data:)
       structured_data.each do |region, records|
         region_id = EducationFacility.facility_for(region: region)
-        filename = "#{region_id}_#{Time.zone.today.strftime('%F').tr('-', '_')}_vetsgov.spl"
+        filename = "#{region_id}_#{Time.zone.today.strftime('%m%d%Y')}_vetsgov.spl"
```

Comment:
do we want to keep the order the same, just without the underscores? so `%Y%m%d`?